src/backend/storage/ipc/sinvaladt.c

```diff
@@ -102,96 +102,9 @@
  * barrier instructions that are included in the spinlock sequences.
  */
 
+SISeg *shmInvalBuffer = NULL;
 
-/*
- * Configurable parameters.
- *
- * MAXNUMMESSAGES: max number of shared-inval messages we can buffer.
- * Must be a power of 2 for speed.
- *
- * MSGNUMWRAPAROUND: how often to reduce MsgNum variables to avoid overflow.
- * Must be a multiple of MAXNUMMESSAGES.  Should be large.
- *
- * CLEANUP_MIN: the minimum number of messages that must be in the buffer
- * before we bother to call SICleanupQueue.
- *
- * CLEANUP_QUANTUM: how often (in messages) to call SICleanupQueue once
- * we exceed CLEANUP_MIN.  Should be a power of 2 for speed.
- *
- * SIG_THRESHOLD: the minimum number of messages a backend must have fallen
- * behind before we'll send it PROCSIG_CATCHUP_INTERRUPT.
- *
- * WRITE_QUANTUM: the max number of messages to push into the buffer per
- * iteration of SIInsertDataEntries.  Noncritical but should be less than
- * CLEANUP_QUANTUM, because we only consider calling SICleanupQueue once
- * per iteration.
- */
-
-#define MAXNUMMESSAGES 4096
-#define MSGNUMWRAPAROUND (MAXNUMMESSAGES * 262144)
-#define CLEANUP_MIN (MAXNUMMESSAGES / 2)
```

Comment:
I saw a lot of codes like this pr, to make some type public.
How about adding a `#if 0` for PG codes if we just want move them to another file? It will save a lot when merging PG with less conflicts.